lowranceusr.cc

```diff
@@ -139,7 +140,7 @@ LowranceusrFormat::register_waypt(const Waypoint* wpt) const
   }
 
   if (global_opts.debug_level >= 2) {
-    printf(MYNAME " adding waypt %s (%s) to table at index %d\n",
+    printf(MYNAME " adding waypt %s (%s) to table at index %" PRId64 "\n",
```

Comment:
This might be a problem in Qt on MacOS. The warning I'm trying to fix was:
lowranceusr.cc:144:70: warning: format specifies type 'int' but the argument has type 'qsizetype' (aka 'long long') [-Wformat]
but if we look at the doc at https://doc.qt.io/qt-5/qlist.html#length we see they really are just plain ints.

Maybe we just cast these to plain ole ints and accept that we're going to shorten what we're printing, but that's OK because if we had > 4.2bn (or the square of that for 64-bit) we're going to have way too much debug print action at best and run out of memory soemwhere else; it's not like you're going to pass a 4.2bn geojson file to a lowrance...

Reply:
Be aware of the Qt 6 change "QList's (and hence QVector's) size type is changed from int to qsizetype."

https://doc.qt.io/qt-6/qtcore-changes-qt6.html#qvector-qlist